relay/relay.go

```diff
@@ -413,7 +417,7 @@ func (r *Relay) handleEvents(ctx context.Context, reader2 reader.Reader, transfo
 		relayLogWriteSizeHistogram.Observe(float64(e.Header.EventSize))
 		relayLogPosGauge.WithLabelValues("relay").Set(float64(lastPos.Pos))
 		if index, err2 := binlog.GetFilenameIndex(lastPos.Name); err2 != nil {
-			log.Errorf("[relay] parse binlog file name %s err %v", lastPos.Name, err2)
+			r.logger.Error("parse binlog file name", zap.String("file name", lastPos.Name), log.ShortError(err2))
```

Comment:
Is `zap.Error` better for these not propagated errors?

Reply:
`zap.Error` always better, except for too much content. 
output the short error information is enough here if we do it in previous version.
we shouldn't have to adjust them case by case now

Reply:
👌